### DISCUSSION TITLE
Website having redirect issues

### DISCUSSION THREAD

**Discussion by ajmas** [ueberdosis/tiptap#3308] [General Questions] [answered]
I was wanting to check the docs on the [main website](https://www.tiptap.dev/installation), but it is running into a redirect issue.

I tried with both Chrome and Safari on macOS.

<img width="789" alt="image" src="https://user-images.githubusercontent.com/682269/195681000-d065a1f7-7778-4ca8-b18b-725f91040c95.png">

**Comment by james-william-r** [comment]
Just opened an Issue for this - #3314

**Comment by patrickbaber** [comment, marked as answer]
Thanks for reporting the bug in our redirect rule. Fixed https://github.com/ueberdosis/tiptap/issues/3314#issuecomment-1280603351
